SequenceAnalysis/src/org/labkey/sequenceanalysis/pipeline/VariantProcessingJob.java

```diff
@@ -408,9 +410,9 @@ public void intervalSerializeTest() throws Exception
         {
             VariantProcessingJob job1 = new VariantProcessingJob(){
                 @Override
-                public File getDataDirectory()
+                public FileLike getDataDirectoryFileLike()
                 {
-                    return new File(System.getProperty("java.io.tmpdir"));
+                    return FileSystemLike.wrapFile(new File(System.getProperty("java.io.tmpdir")));
```

Comment:
Hmm, all these usages of System.getProperty("java.io.tmpdir") should be replaced with some call to FileUtil.  For the TODO list.

Reply:
Yes, unless the code actually depends on using this temp directory vs whatever path LabKey has chosen as its default (might be the same, I didn't dive in).

Reply:
Two comments:
- In general, I dont see any real problem with LabKey enforcing a layer over the filesystem, and it's probably an overdue and good idea. In experimented with migrating code in 25.7, and found the FileUtil helpers associated with FileLike to be kinda cumbersome; however, it seems like you're doing a lot on this front in the current release.
- I cannot think of any reason our modules specifically need java.io.tmpdir. That's just the standard way, to my knowledge, for java code to find the tempdir. Having LabKey introduce a layer over this is probably a good idea.